content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,6 +1,6 @@
-===============
-Closing tickets
-===============
+=============
+Close tickets
+=============
 
 Once work has been completed on a *Helpdesk* ticket in Odoo, there are several ways it can be
 closed. Manually closing solved tickets keeps the pipeline up to date, while automatically closing
```

Comment:
Just curious- how does not automatically closing tickets make a blocking issue? Wouldn't this just have open ticket numbers remain high? I thought a blocking issue was something stopping something form happening- so I don't see this as blocking, but I don't use Helpdesk, so I could definitely be wrong!